src/shared/ui/SearchForm/SearchForm.js

```diff
@@ -12,20 +12,20 @@ export const SearchForm = ({ initialValue = "", onSearch }) => {
     onSearch(searchText);
   };
 
-  const handleKeyPress=(event)=>{
+  const handleKeyPress = (event) => {
     if (event.key === "Enter") {
       onSearch(searchText);
     }
   }
 
   return (
-    <form class="search-form input-element">
-      <input type="text" placeholder="Search...."
+    <form className="search-form input-element">
+      <input type="text" data-testid="search-input" data-cy="search-input" placeholder="Search...."
         value={searchText}
         onChange={handleInputChange}
         onKeyDown={handleKeyPress}
       />
-      <button type="button" onClick={handleSearchClick}>Search</button>
+      <button type="button" data-cy="button-input" onClick={handleSearchClick}>Search</button>
```

Comment:
```suggestion
      <button type="submit" data-cy="button-input" >Search</button>
```

https://learn.javascript.ru/forms-submit

Reply:
changed